app/views/root/index.html.erb

```diff
@@ -48,9 +48,6 @@
 
       <section class="services-block" aria-labelledby="services-and-information-label" id="services-and-information">
         <div class="inner-block floated-children">
-          <div class="two-column-heading">
-            <h2 id="services-and-information-label">Services and information</h2>
```

Comment:
Had a quick offline chat with John Turnbull, Charlotte Clancy and Mark Hurrell - we couldn't come up with anything which is an improvement, and it's not a terrible description of the following links, so we're going to keep the wording as it is, and just visuallyhide it.